### PR TITLE
Add Disguise to list of supported applications

### PR DESCRIPTION
**Pull request by chrisnashdisguise**
Replaces https://github.com/AcademySoftwareFoundation/OpenColorIO/pull/2046

We at Disguise have recently added support for OpenColorIO to our Designer software, due for release on September 25th. Please could we add Disguise to the list of supported applications on the OpenColorIO homepage.

### CODE REVIEW

File: docs/site/homepage/data/en/supported_apps.yml [hunk 1]
```diff
@@ -58,6 +58,13 @@ supported_apps:
       categories : ["Game Engine", "3D"]
       content : Version 3 and up.
       link : "https://www.cryengine.com/"
+      
+    # portfolio item loop
+    - name : Disguise
+      image : images/supported_apps/disguise.png
+      categories : ["2D", "3D", "Compositing", "Playback", "Simulation"]
```
Comment (doug-walker) [inline docs/site/homepage/data/en/supported_apps.yml:65]: I'm wondering if we should add a "Virtual Production" category?  Any thoughts @carolalynn ?

**Review by doug-walker — APPROVED** [review]
Thanks Chris, congrats on adding support, we're glad to have it on the website!

**Review by zachlewis — APPROVED** [review]

**Comment by chrisnashdisguise** [comment]
Hi @doug-walker , I've been keeping an eye out but it seems like these changes haven't made it onto the website yet. Did I do something wrong, or does the site need an update somehow? Thanks!

**Comment by doug-walker** [comment]
@chrisnashdisguise , apologies for the delay. We've had issues with the system that deploys the website. We are working on improving this. I will try and ensure this gets fixed over the next week.

**Comment by chrisnashdisguise** [comment]
Ok no problem, thanks for the update!

**Pull request merged**